notebooks/240-efficient-object-detection/240-efficient-object-detection.ipynb

```diff
@@ -0,0 +1,612 @@
2NIaOUe2FrjPHSt1eO3HjKXk7ZMyglfd7ppGYaLCBfIxSgRB5v3lpOJmNWV1bxARFwKQaxFdFAhKqNkercSWSmbixlkQtpwSiKrOjE79tOtN+gNFgrJKvhcIh3Ed1Qi8NxkbzR3WM52Ey0g0wELBHccLGS8X7BaC/LsjvcOmEMFn2/5ZGllLAtE2pSUF+unrtnF9EapQxGL1y+0mctikK00SMkqfSSGhfgvBWGu7U46+WAURkqIG4+racKbezRS4C1VhKMum1pmpq2FSKhBnq9kjJT5DiKwsR+FTx8//3cf9+9XLr7Ls6c2ZFWTK4wyuF9g0U8wJu55dc++xscnYwpguFUUXB20GdjZ5uJ8ZxdXeOBrUscjk+4dvuQrz39El/++lM89tGPAke41SH9c3dh65rDW3tsDNc4vr7HKBjm4zGj0YAqh9n2ClfG+5xeu4Afj9F7J6wczimVxinPRMM3r7xFk3l+3/d9P1uDDf7Gt17hiY99kjf33sTMao6Op5gMHnzwIi9WDXu7h5y/cIHd2R4XVlfwvYI6z7lytMtcG1orFqmEILaEcfxosdaE85IOxiSGY50DrfDao1TWiQ35kM6UNCq30Jxe5iYoLwE59aIT1JzWU9u2XWtF2lyiirUs7yvrVEfC1gIuFalZSeKM1hTDISsrI554/DF2zpzhH/7sP6JtLc75TgP/9Jkz7B0ecubsecbTCfPZTOZljUEZzUV/kba1XLnyNtubWyhkCuX6zRtxDlnTNg29sidVpY898Nh2m87m9AZ9MqM5e2YHo6Xzmsb+vA9Y78lAfAraKX/sj/4hvviVr3Owv8vW5gb7Bwfcun2ICxn/7Bd/lT/9b/8EyskYpI9BT2uNCpmMZuFBhY7oq5RCGWkXJp/pXFlQgbxvGPbWOH9mg0cfusTJdMbu7iFvvHmDF195nf3jGdoUuGDwRAvMqHnhw50B+J1VrPxbSpRSWyNVykGCOioifJbBQExwkmRzcvZb5iD9bte7n0PWSbfZolByQOBiL1cRgukOT2MWTGhr6yhV6fBeMhNh11qCc2IO71qcFyN7HZz0GXMTfXwhywXWjL9+RA40ZdHHWiss6WhRqJSIkIjdnu6UYZLWqbSDZOzKO/m6NiJjuTjQF+5Ny5B1urHpZgtUvoBO5ZJgncQmlh9293O2hehKlIQ4ZDMEdDSlt60V6cLWYkzAKEev0AwGPbJMBDOm0wnHxxVNA641QJ8QMlrrcG6OzhRK+7jZcxQBpaI+qwuA+Ci3TUOWC2sYt2CnC0wvVX9WSLugblu8ihaEWQbeUpZFFxxTsCvKvFuMSSBFEgViVZp3/sxp8XcHjRexDwnEjiwNHwYiIWxhvL6cRC2TMtK/p7+/k6SzDCktM2RF2zejaRu88504S4I+lx5zl0jJqJOodw1HI0xmMMbRL0V0pmnFq7Wez2mtFehUxb4W0DqpvJOBhwoS/HXwrI362HpCdXLIffffw3/wH/4F+mVGZlScI3Z4N0dHJCVohVeGzPR4+bUXePmtK7RGsbky4IGL53FhTrVVcnR8xIUsZ//pb6J7fe6/5z6eu3KNF994g7W7LsrnysBp0MdTzk4D+uAAO69pBwXFmU1YHdJfHzJ1LVt6k/zmMfb2MZkVlv848/hTO3z2a89QG80HP/QIWXXCZ3/5C7xn+25+8Pd8it7d6/xv/8vf5u2r13jjzbd5z3sf5bXXr9MeTtnbO2Tnrh1uzmYU505zfTLlprfMERKkjkbRzraYIJC1N0tysfE5J7U7AJNJ60VFYZk0s54OXkizwKYLvMsQuGEhBOKjgX0IgTYq8sXFtfgjdG29lPylCvnOyikaXIAUHs5S5IaHH364cxN76KGHePbZZ6WQwGN0zsHRIf1+n/5oyMHhIVlEAa5ducL7P/hBNre3WR2tYrTBtZbpZMJoMCDLNEWR0ev1mE4m1M08opmKXtQfUEazvr5Br9djPN7nrrsuUOQ5mfIYbePvJON62oDyHrTnrgtn+CN/5Mf4Oz/zj7n/wUc4ODjg7Ws3eeThe/n8b36JH/z093DvxfME14KWtgBKrCFlQsdLhaxEItT5EBUcDdrUQIMyJsoGO8oMCh+wrqEcGAan1ygIFCbw9POvczQTdroNwg9Ir02Ssl16Xu+8kgGGINqKJPPcBepouLM8sZEEnBI6ujyC+7td797tyYALDqWlkpODU+T20jhB07bCiNUC5aCEfKIBnWdxbCfgrGSL/X4ppJVY+meJ2JXr6GIkVYSKMDZI704UuRaeost9waRUtSwMDt+O46fq17loi21Up0hlogZ2CropKCc1sGS5twyDpoBk25aUQVnbxtEZObgHg4HA1spHNSSh0ddNBcqjnEdrDzh6fcN8cpWtnQ2MznEt2LqlmrbMpnOMzvE+Q2WnGPRKKiqs89Rtg841RU96sUpryrzH7KQWYChAE8VPki1DrxCyEl2AyUUgRSVFNaIPs2TGTQxSMktsaNs63gcZITFm4bIlh1dywkmVaqwU3KK6kD51TmYMlgX7WSpheYbOLYRM0v/TtSwGL2YL0gJYrpyXCXqpUumsNbt1kXqHkk2n9SR7VjawKD5J5SOfU+Av5z1nzuywvrZOU+9hC0m0FIG6abBtQ13XsolViEiDjr1Lybjjzmc+OSHPDCcGellgZVjy7/+5P80g1+AFmcizCNGrIIhCazEhI+8PefHVK3zuq08xqVrOnzrFA/fczdF4l7XtIbn13O007uW32Paeo3nDbLjHfRcvcvDi67z60quwcz8bWZ/6jcuUR1MKleOCYri9Btur1MZw6/YhW+S0N/dYcxp3OGE4HLCf19hzmxwEzde/8TrWF/zo936Utjrka5/5Im++ucujjzzBrbefZ8Wc4w//mz/GX/vrf53Pf/EpGvdlQnBs7mxxY+8WF8+e4gtf+homePpbm0wCWOsxyhC0pw4WZyLDWi0QC0mi9dIajIiXFXlE7xRZprufwbqurSQ9xIUgzzKKwt
```

Comment:
Please mention that the first-time data download might take time and downloading speed depends on internet connection.